features/specific/predicate_message.feature

```diff
@@ -0,0 +1,72 @@
+@boost
+Feature: Predicate Message Feature
```

Comment:
Either this should be in a `boost` directory or it should specify that it is related to Boost. Please remove the redundant `Feature` at the end.